test/functional/test_framework/blocktools.py

```diff
@@ -128,6 +133,22 @@ def sign_coinbase(node, coinbase):
     return coinbase
 
 
+def get_finalization_rewards(node, current_height=None):
+    epoch_length = node.getfinalizationconfig()['epochLength']
+    if not current_height:
+        current_height = node.getblockcount()
+    if current_height % epoch_length != 0 or current_height == 0:
+        return []
+    result = []
+    for i in range(epoch_length):
+        block_hash = node.getblockhash(current_height - i)
+        coinbase = node.getblock(block_hash, 2)['tx'][0]
+        reward_out = coinbase['vout'][0]
+        reward_script = CScript(hex_str_to_bytes(reward_out['scriptPubKey']['hex']))
+        result.append(CTxOut(FULL_FINALIZATION_REWARD * UNIT, reward_script))
+    return result[::-1]
```

Comment:
Instead of inverting the result, you could invert the range used for the `for` loop.

Reply:
Replaced with `for i in range(epoch_length)[::-1]`